utils/webpack/src/js/hoverHighlight.js

```diff
@@ -0,0 +1,39 @@
+export function getKeywords(){
+    var keywordList = document.getElementsByTagName("highlight-term")
+
+    for (let i = 0; i < keywordList.length; i++) {
+        build_eventlistener(keywordList[i].id, keywordList[i].dataset.label, keywordList[i].dataset.line)
+    }
+}
+
+
+export function build_eventlistener(command_id, code_box_label, code_line_number){
+    // get the dynamic variables created by Hugo shortcode
+    // var command_id = document.currentScript.getAttribute('rand_id');
```

Comment:
Do we need to include this commented out code?

Reply:
Removed!